requirements/mypy.in

```diff
@@ -2,4 +2,7 @@
 # /tools/update-locked-requirements to update requirements/dev.txt
 # and requirements/mypy.txt.
 # See requirements/README.md for more detail.
-mypy==0.670
+mypy==0.700
+# Include typing explicitly, since it's needed on Python 3.4
+typing==3.6.6
+typing_extensions==3.6.6
```

Comment:
We don’t support Python 3.4 anymore, so we shouldn’t need this, right?

Reply:
Correct.

Reply:
Gotcha, removed.